src/pages/index.js

```diff
@@ -44,11 +60,16 @@ avatarFormValidator.enableValidation(); // call enable validation
 // =========  POPUP CLASS INSTANCES ========= //
 
 // New Popup Form for adding a card
-const newCardPopup = new PopupWithForm("#add-card-modal", (formData) => {
-  const name = formData.title;
-  const link = formData.link;
-  handleNewCardSubmit(name, link);
-});
+const newCardPopup = new PopupWithForm(
+  "#add-card-modal",
+  (formData, renderLoading) => {
+    const name = formData.title;
+    const link = formData.link;
+    handleNewCardSubmit(name, link, () => {
+      renderLoading();
+    });
```

Comment:
`renderLoading` is not needed here. It's used inside `handleNewCardSubmit` differently

So, here will be only

```js
  (formData) => {
    const name = formData.title;
    const link = formData.link;
    handleNewCardSubmit(name, link);
  }
```

Reply:
thank you